KEGG_parser/__init__.py

```diff
@@ -0,0 +1 @@
+__version__ = '0.0.5'
```

Comment:
```suggestion
__version__ = '0.0.1'
```

Reply:
To restart versioning for this. I generally try to stick with semvar and with this type of package that is not standalone I think it would be good. I already have projects that I think could use this in my new lab.